CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# Upcoming
+
+* Strip down forms sass to be simpler and more useful
```

Comment:
Do we need to further elaborate on what `more useful` is. To me it doesnt mean much, unless i look at the sass